### DISCUSSION TITLE
Jetson Nano 4GB

### DISCUSSION THREAD

**Discussion by artyomboyko** [NVIDIA-NeMo/NeMo#2116] [General]
Hello. Can someone tell me. Can I use neno docker container in Jetson Nano?

**Comment by okuchaiev** [comment]
we currently do not test/support NeMo on Jetson. Feel free to try and share your experience.

**Reply by artyomboyko** [reply]
Добрый день. Не вопрос. Но я новичек во всем этом. У меня есть Jetson Nano 4 ГБ + JetPack 4.5.1. + Logitech C920. Хочу попробовать превратить его в чатбот. С чего бы начать?
